### ISSUE TITLE
Left ventricular dilatation

### ISSUE THREAD

**Issue by sng2140** [obophenotype/human-phenotype-ontology#9585]
There is specification for the Right but not left or both

**Preferred term label:**
Left ventricular dilatation

**Synonyms**
Left ventricular dilation; dilated left heart ventricle

**Definition (free text, please give PubMed ID)**
Enlargement of the chamber of the left heart ventricle.

**Parent term (use hpo.jax.org/app)**
Abnormal left ventricle morphology

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
added as HP:4000141
